core/src/main/resources/org/jboss/jbossset/bugclerk/JIRAMatchStreamForIncorporated.drl

```diff
@@ -0,0 +1,45 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2015, Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags. See the copyright.txt file in the
+ * distribution for a full listing of individual contributors.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+
+package org.jboss.jbossset.bugclerk;
+
+import org.jboss.set.aphrodite.domain.Issue;
+import org.jboss.set.aphrodite.domain.Comment;
+import org.jboss.set.aphrodite.issue.trackers.jira.JiraIssue;
+
+import org.jboss.jbossset.bugclerk.Violation;
+import org.jboss.jbossset.bugclerk.Severity;
+
+global java.util.List list
+
+rule "JIRAMatchStreamForIncorporated"
+  salience 0
+  dialect "mvel"
+  when
+    $parent : Candidate(JIRA, !(((JiraIssue)bug).getLinkedIncorporatesIssues()).isEmpty(), $children: ((JiraIssue)bug).getLinkedIncorporatesIssues() ,filtered == false);
```

Comment:
For the cast part, I think [there is a somewhat](https://github.com/jboss-set/bug-clerk/blob/master/core/src/main/resources/org/jboss/jbossset/bugclerk/PRLinksNotValid.drl#L39) cleaner (arguably) syntax, I would rather use this one over the one here.